torchvision/prototype/models/vision_transformer.py

```diff
@@ -2,18 +2,13 @@
 # https://github.com/google-research/vision_transformer
 # https://github.com/facebookresearch/ClassyVision/blob/main/classy_vision/models/vision_transformer.py
 
-import math
-from collections import OrderedDict
 from functools import partial
-from typing import Any, Callable, Optional
+from typing import Any, Optional
 
-import torch
-import torch.nn as nn
-from torch import Tensor
 from torchvision.prototype.transforms import ImageNetEval
 from torchvision.transforms.functional import InterpolationMode
 
-from ...utils import _log_api_usage_once
+from ...models.vision_transformer import VisionTransformer
```

Comment:
Let's import `interpolate_embeddings` here so that we expose on the prototype the same methods as on the main area. This will make transitioning between the two APIs easier:
```suggestion
from ...models.vision_transformer import VisionTransformer, interpolate_embeddings
```

If you get linter errors due to the unused import, feel free to turn off.